1753683755292-30b3431f487b4cc1863e57a81d78e289.sh

```
@@ -0,0 +1,204 @@
+            log_error "Failed to create directory: $dir"
+            exit 1
+        }
+    fi
+}
+
+# ========================
+#     Node.js 安装函数
+# ========================
+
+install_nodejs() {
+    local platform=$(uname -s)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)